### PR TITLE
Adds builds history tracking

### PR DESCRIPTION
**Pull request by d4rkr00t**

### CODE REVIEW

File: src/Runner.js [hunk 1]
```diff
@@ -271,15 +242,17 @@ export default class Runner {
...
       running: Object.assign({}, this.running),
       waitingToLand: this.waitingToLand,
       locked: this.locked,
-      started: this.started,
+      started: String(this.started),
       paused: this.paused,
-      pausedReason: this.pausedReason
+      pausedReason: this.pausedReason,
+      history: this.history.take(10),
+      usersAllowedToMerge: this.config.prSettings.usersAllowedToApprove
```
Comment (d4rkr00t) [inline src/Runner.js]: I added that back, need it in UI :)

File: src/hosts/GitHubAdapter.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // @flow
 import { type HostAdapter, type StatusEvent, type LandRequest } from '../types';
 
+// $FlowFixMe
```
Comment (d4rkr00t) [inline src/hosts/GitHubAdapter.js]: ¯\_(ツ)_/¯

File: src/Runner.js (hunk 1 above)
Comment (lukebatchelor) [reply src/Runner.js]: We can fetch this with the settings endpoint.
Will do that in a follow up though I think.

File: src/Runner.js [hunk 3]
```diff
@@ -48,25 +53,15 @@ export default class Runner {
     let landRequest: ?LandRequest = this.queue.dequeue();
     if (!landRequest) return;
     this.locked = true;
-    Logger.info(
-      {
-        landRequest
-      },
-      'Checking if still allowed to land...'
-    );
+    Logger.info({ landRequest }, 'Checking if still allowed to land...');
```
Comment (lukebatchelor) [inline src/Runner.js]: Hmmm.. I need to figure out why my prettier seems to be changing these and yours isnt?

File: src/Runner.js [hunk 4]
```diff
@@ -94,77 +86,56 @@ export default class Runner {
...
-    Logger.info(
-      {
-        pullRequestId,
-        running
-      },
-      'Merging pull request'
-    );
-    this.client.mergePullRequest(pullRequestId);
+    Logger.info({ pullRequestId, running }, 'Merging pull request');
+    // this.client.mergePullRequest(pullRequestId);
```
Comment (lukebatchelor) [inline src/Runner.js]: I'll uncomment this once its all merged in ;P

Comment (d4rkr00t) [reply src/Runner.js]: 🤦‍♂️

File: src/Runner.js (hunk 1 above)
Comment (d4rkr00t) [reply src/Runner.js]: Oh, right, you told me about settings endpoint, and I totally forgot about it :( sorry

**Pull request merged**